boilerplates/react/panda.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from "@pandacss/dev";
+
+export default defineConfig({
+  // Whether to use css reset
+  preflight: true,
+
+  // Where to look for your css declarations
+  include: [
+    "./components/**/*.{js,jsx,ts,tsx}",
+    "./layouts/**/*.{js,jsx,ts,tsx}",
+    "./pages/**/*.{js,jsx,ts,tsx}",
+    "./src/**/*.{js,jsx,ts,tsx}",
+  ],
+
+  // Files to exclude
+  exclude: [],
+
+  // Useful for theme customization
+  theme: {
+    extend: {},
+  },
+
+  // The output directory for your css system
+  outdir: "files/styled-system",
+});
```

Comment:
Is this necessary? We already have `boilerplates/panda-css/files/panda.config.ts`

Reply:
Yes, it is. Alongside with `@pandacss/dev` package and `"prepare": "panda codegen --silent",` script inside `package.json`, PandaCSS will generates `styled-system` dir inside `./files` when installing packages (`pnpm install`). So that we can add usages of PandaCSS : `import { css } from "../styled-system/css";`

The `styled-system/` directory is generated automatically and added to `.gitignore`, so AFAIK, we can't use it by importing from the `panda-css` boilerplate, since it's not included in the build.

Reply:
But I think we can minimize this config file, let me try

Reply:
I see. We should not call the script `prepare` on each boilerplate IMO, this prepare script should probably only be added to generated `package.json` files.
That way we do not need to codegen them inside boilerplates, but they will be generated when users `pnpm install` the generated project.

Reply:
But the boilerplates need it for example usage. I also did it like this so that the imports is correct when users scaffold it

Reply:
:+1: I see. With latest changes it's not that big of a deal anymore